api/ooniapi/incidents.py

```diff
@@ -286,6 +306,8 @@ def post_update_incident(action: str) -> Response:
         if action == "create":
             incident_id = str(generate_random_intuid(current_app))
             req["id"] = incident_id
+            if get_client_role() != "admin" and mismatched_email_addr(req):
+                raise InvalidRequest
             if get_client_role() != "admin" and req["published"] == 1:
```

Comment:
Shouldn't there be a check for `mismatched_email_addr` in here as well?

Reply:
In which line exactly? The check is being done on creation and update and not on deletion because, well, it's a deletion.